app/bundles/CoreBundle/Command/SendChannelBroadcastCommand.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * @package     Mautic
+ * @copyright   2016 Mautic Contributors. All rights reserved.
+ * @author      Mautic
+ * @link        http://mautic.org
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\CoreBundle\Command;
+
+use Mautic\CoreBundle\CoreEvents;
+use Mautic\CoreBundle\Event\ChannelBroadcastEvent;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * CLI Command to send a scheduled broadcast
+ */
+class SendChannelBroadcastCommand extends ModeratedCommand
+{
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this->setName('mautic:send:broadcasts')
```

Comment:
I think we need to settle on a command structure for simplicity and easy-to-guess-ish-ness. 

I checked, and even in our current commands we aren't consistent (although 2.0 brought us closer). 

I'd like to have it be `mautic:NOUN:VERB`, so in this case, `mautic:broadcasts:send`